rinks/wheaton.html

```diff
@@ -0,0 +1,368 @@
+									</td>
+
+									<td>
+										<span class="date">11</span>
+
+									</td>
+
+									<td>
+										<span class="date">12</span>
+
+									</td>
+								</tr>
+
+								<tr>
+									<td>
+										<span class="date">13</span>
+
+									</td>
+
+									<td>
+										<span class="date">14</span>
+
+									</td>
+
+									<td>
+										<span class="date">15</span>
+                    <div class=calTime>
+                      SnS</br>1:15pm - 2:25pm
+                    </br>4:00pm - 5:20pm
+                    </div>
+									</td>
+
+									<td>
+										<span class="date">16</span>
+
+									</td>
+
+									<td>
+										<span class="date">17</span>
+                    <div class=calTime>
+                      SnS</br>1:15pm - 2:25pm
+                    </br>4:00pm - 5:20pm
+                    </div>
+									</td>
+
+									<td>
+										<span class="date">18</span>
+
+									</td>
+
+									<td>
+										<span class="date">19</span>
+                    <div class=calTime>
+                      SnS</br>1:15pm - 2:25pm
+                    </br>4:00pm - 5:20pm
+                    </div>
+									</td>
+								</tr>
+
+								<tr>
+									<td>
+										<span class="date">20</span>
+                    <div class=calTime>
+                      SnS</br>1:15pm - 2:25pm
+                    </br>4:00pm - 5:20pm
+                    </div>
+									</td>
+
+									<td>
+										<span class="date">21</span>
+
+									</td>
+
+									<td>
+										<span class="date">22</span>
+                    <div class=calTime>
+                      SnS</br>1:15pm - 2:25pm
+                    </br>4:00pm - 5:20pm
+                    </div>
+									</td>
+
+									<td>
+										<span class="date">23</span>
+
+									</td>
+
+									<td>
+										<span class="date">24</span>
+                    <div class=calTime>
+                      SnS</br>1:15pm - 2:25pm
+                    </br>4:00pm - 5:20pm
+                    </div>
+									</td>
+
+									<td>
+										<span class="date">25</span>
+
+									</td>
+
+									<td>
+										<span class="date">26</span>
+
+									</td>
+								</tr>
+
+								<tr>
+									<td>
+										<span class="date">27</span>
+
+									</td>
+
+									<td>
+										<span class="date">28</span>
+
+									</td>
+
+									<td>
+										<span class="date">29</span>
+                    <div class=calTime>
+                      SnS</br>1:15pm - 2:25pm
+                    </br>4:00pm - 5:20pm
+                    </div>
+									</td>
+
+									<td>
+										<span class="date">30</span>
+
+									</td>
+								</tr>
+
+
+						</table>
+					</div> <!--end calendar-->
+			</div> <!--end right column-->
+
+		</div> <!--end container-->
+
+
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="js/index.js"></script>
+  </body>
+  </html>
```

Comment:
Really great job overall. I like the organization of the different pages, the templating, the styles. It's all really coming together. Keep up the good work and let me know if you have any questions. 

Merge this whenever you're ready!